drivers/lora/Kconfig.sx12xx

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2019 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_SX1276 = semtech,sx1276
```

Comment:
Please add SEMTECH_ before each, for consistency with the rest of the Kconfig code.

You may also want to consider using `:=` instead of `=`. The former expands once, the latter each time the variable is read (kind of like Make's two types of variables). Since you have no need for re-expansion it would be clearer to use `:=`.

Like this:

```
DT_COMPAT_NORDIC_NRF_SPI  := nordic,nrf-spi
```

But instead:

```
DT_COMPAT_SEMTECH_SX1276  := semtech,sx1276
```

etc.

Reply:
Ack